src/scripts/RecipeCard.js

```diff
@@ -1,6 +1,6 @@
 import { searchForKey } from './searchKey.js';
 import { markFav, unFav } from './FavoriteRecipe.js';
-import { deleteRecipe } from './deleteRecipe.js';
+//import { deleteRecipe } from './deleteRecipe.js';
```

Comment:
Why are we doing this? afaik recipecard.js's deletebutton needs to know that deleteRecipe() exists